Index: lmbench-3.0-a9+debian.1/scripts/config-run
===================================================================
--- lmbench-3.0-a9+debian.1.orig/scripts/config-run
+++ lmbench-3.0-a9+debian.1/scripts/config-run
@@ -626,13 +626,7 @@ Please specify a directory that has enou
 system.
 
 EOF
-	DEFAULTFSDIR=/var/tmp
-	for t in /var/tmp /tmp; do
-		if [ -d $t -a -w $t ]
-		then	DEFAULTFSDIR=$t
-			break
-		fi
-	done
+	DEFAULTFSDIR=/var/tmp/lmbench
 	echo $ECHON "FSDIR [default $DEFAULTFSDIR]: $ECHOC"
 	read FSDIR
 	if [ X$FSDIR = X ]
